extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/VertxCoreRecorder.java

```diff
@@ -12,6 +12,8 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import org.wildfly.common.cpu.ProcessorInfo;
```

Comment:
Vert.x has something very similar. As this one does not add a dependency, we can keep this one. If one day the dependency is removed, we have a fallback.